unittests/abi_tests.cpp

```diff
@@ -2054,8 +2054,7 @@ BOOST_AUTO_TEST_CASE(abi_account_name_in_eosio_abi)
    }
    )=====";
 
-   auto abi = eosio_contract_abi(fc::json::from_string(repeat_abi).as<abi_def>());
-   auto is_type_exception = [](fc::assert_exception const & e) -> bool { return e.to_detail_string().find("abi.types.size") != std::string::npos; };
```

Comment:
Maybe here we should report error if `is_type_exception` returns `true`, instead removing it?

Reply:
That is a lambda that is not used. It is copy/pasted from other tests where it is actually used.